config/cloud.cfg.tmpl

```diff
@@ -124,47 +133,44 @@ cloud_config_modules:
  - locale
 {% endif %}
  - set_passwords
+{% if variant == "alpine" %}
```

Comment:
Why'd you move this block? In general I don't think we should be reordering modules unless there's a really good reason.

Reply:
Alphabetising the distro's package managers :-)

Obviously package managers are exclusive of each other (if, elif, elif etc) so reordering does matter.